content/projects/first-personal-website/_index.md

```diff
@@ -64,4 +64,12 @@ Also, we'll be looking out for good developer habits. Please make sure you push
 - [More info on CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/)
 - [Web Developer Portfolios to inspire you](https://medium.freecodecamp.org/15-web-developer-portfolios-to-inspire-you-137fb1743cae)
 - [Building a Data Science Portfolio](https://www.dataquest.io/blog/build-a-data-science-portfolio/)
-- [Tips to include towards creating an effective Portfolio](https://www.freecodecamp.org/news/how-to-build-an-awesome-data-science-portfolio/)
\ No newline at end of file
+- [Tips to include towards creating an effective Portfolio](https://www.freecodecamp.org/news/how-to-build-an-awesome-data-science-portfolio/)
+
+## Instructions for reviewer 
+
+- The link submitted should direct you straight to the website
+- Everything should be linked correctly [social media accounts, work profiles like (github,linkedIN), CV, portfolio/projects,pages] 
+- Font and text size should be very clear [makes reading efficient]
+- They should make use of the entire web page [work spaced out and not squashed]
```

Comment:
Please revise Engtlish and brackets. 

Use round brackets in text. Square brackets have specific meanings in markdown and code. It's better to use normal english stuff

Also some sentence structure can be made a bit better. Consider using something like Grammerly going forward. 

This is a hell of a sentence:
```
- Everything should be linked correctly [social media accounts, work profiles like (github,linkedIN), CV, portfolio/projects,pages] 
```

I tried to fix the punctuation but it was much easier just to write a more concise sentence:

- All links should work

Also, it would be good to know a bit more about how brackets work in English. Some of the things you did are odd.

This is bad bracketing: `blab blah like (things, stuff) blah`
This is better: `blah blah (for example, things and stuff) blah`